### ISSUE TITLE
Mercedes has changed their API, not able to sign in anymore

### ISSUE THREAD

**Issue by Martinezgrande** [SeydX/homebridge-mercedesme#59]
Mercedes has changed their API. 

Need to create new credentials 

Also new auth URL and Token URL

Auth URL
old URL: https://id.mercedes-benz.com/as/authorization.oauth2
new URL: https://ssoalpha.dvb.corpinter.net/v1/auth

Token URL
old URL: https://id.mercedes-benz.com/as/token.oauth2
new URL: https://ssoalpha.dvb.corpinter.net/v1/token

Also new scope openid needs to be added

And app needs to support new JWT token format

**Comment by phantomgnom** [comment]
Hi, looks like, the plugin is useless for now. Very sad to see the plugin does not work anymore. I am pretty sure sydex spend a lot of time to make it work. It did, until Mercedes did some changes to their Api. I wanted to automate my garage opener not to start of the trunk lit is open. @seydx : What kind of sponsorship is required to help you updating accordingly?

**Comment by Martinezgrande** [comment]
> Hi, looks like, the plugin is useless for now. Very sad to see the plugin does not work anymore. I am pretty sure sydex spend a lot of time to make it work. It did, until Mercedes did some changes to their Api. I wanted to automate my garage opener not to start of the trunk lit is open. @seydx : What kind of sponsorship is required to help you updating accordingly?

Yes, I would be more then happy to express my gratitude in a financial compensation.

**Comment by phantomgnom** [comment]
Hi Martinez, I had no time to test yet. I will do next week. Do you have paypal or something like that?RegardsTimoAm 31.01.2023 um 13:54 schrieb Martinezgrande ***@***.***>:﻿

Hi, looks like, the plugin is useless for now. Very sad to see the plugin does not work anymore. I am pretty sure sydex spend a lot of time to make it work. It did, until Mercedes did some changes to their Api. I wanted to automate my garage opener not to start of the trunk lit is open. @seydx : What kind of sponsorship is required to help you updating accordingly?

Yes, I would be more then happy to express my gratitude in a financial compensation.

**Comment by Martinezgrande** [comment]
> Hi Martinez, I had no time to test yet. I will do next week. Do you have paypal or something like that?RegardsTimoAm 31.01.2023 um 13:54 schrieb Martinezgrande ***@***.***>:﻿ Hi, looks like, the plugin is useless for now. Very sad to see the plugin does not work anymore. I am pretty sure sydex spend a lot of time to make it work. It did, until Mercedes did some changes to their Api. I wanted to automate my garage opener not to start of the trunk lit is open. @seydx : What kind of sponsorship is required to help you updating accordingly? Yes, I would be more then happy to express my gratitude in a financial compensation. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Hey mate, in not the developer. I was merely echoing your remark that I'm also willing to donate to @seydx to get this plugin to work again

Regard

**Comment by Jeepersfinest** [comment]
Any news?
